modules/oe_list_pages_open_vocabularies/tests/src/Kernel/ListPagesSearchApiConfiguratorTest.php

```diff
@@ -189,7 +189,7 @@ public function testVocabularyAssociationChanges(): void {
     $this->assertNull($facet_1);
     $facet_2 = Facet::load($id_2);
     $this->assertEquals('New label', $facet_2->label());
-    $this->assertEqual(SearchApiConfigurator::MAX_WEIGHT, $facet_2->getWeight());
+    $this->assertEquals(SearchApiConfigurator::MAX_WEIGHT, $facet_2->getWeight());
```

Comment:
Looks like also you have to swap the order of arguments (see the signature difference of method).

Reply:
Change updated on next update.